ecl/data_source_ecl_network_qos_options_v2_test.go

```diff
@@ -0,0 +1,104 @@
+package ecl
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccNetworkV2QosOptionsDataSource_basic(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skip this test in short mode")
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccNetworkV2QosOptionsDataSourceBasic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNetworkV2QosOptionsDataSourceID("data.ecl_network_qos_options_v2.qos_options_1"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_qos_options_v2.qos_options_1", "name", "20M-GA-AZURE"),
```

Comment:
実在するQOSオプションを指定して頂きたいので`10Mbps-BestEffort`をご利用頂けたらと思います。
```suggestion
						"data.ecl_network_qos_options_v2.qos_options_1", "name", "10Mbps-BestEffort"),
```